src/components/common/input/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
It seems like all these common components could be used from axis instead of replicating them here, @kushptlapigarage was there an issue with them?